src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -114,29 +125,253 @@ Template:
+| RF transmit field map                                  | TB1map | Parametric   | In percent units (p.u.). Radio frequency (RF) transmit field maps are REQUIRED to use this suffix irrespective of the method they are related to. Please see the [qMRI appendix](../99-appendices/09-qmri.md) for associated inputs and further details                                                                     |
+| RF receive sensitivity map                                  | RB1map | Parametric   | In percent units (p.u.). Radio frequency (RF) receive sensitivity maps are REQUIRED to use this suffix irrespective of the method they are related to. Please see the [qMRI appendix](../99-appendices/09-qmri.md) for associated inputs and further details                                                                     |
+| Observed signal amplitude map                                | S0map | Parametric   | In arbitrary units (a.u.). For a multi-echo sequence, S0 maps index the baseline signal before exponential (T2\*) signal decay. In other words: the exponential of the intercept for a linear decay model across log-transformed echos. For more information, please see, for example, [the tedana documentation](https://tedana.readthedocs.io/en/latest/approach.html#monoexponential-decay-model-fit). S0 maps are REQUIRED to use this suffix irrespective of the method they are related to. _Associated suffixes:_ T2starmap.       |
+| Quantitative susceptibility map (QSM)                   | Chimap | Parametric   | In parts per million (ppm). QSM allows for determining the underlying magnetic susceptibility of tissue (Chi).  Chi-maps are usually constructed from the phase images of GRE sequences. Chi-maps are quantitative and are reconstructed by solving the magnetic field to susceptibility source inverse problem ([Wang & Liu, 2014](https://onlinelibrary.wiley.com/doi/10.1002/mrm.25358)). |
+
+Quantitative maps can be obtained right off the scanner or by processing files
+belonging to a `grouped scan collection`. Regardless of the method they are
+obtained by, one of the `_suffix` entries listed in the table above is REQUIRED
+for a proper naming. For example:
+
+```Text
+sub-01_T1map.nii.gz
+sub-01_T1map.json
+```
+**Important:**
+
+Quantitative unit of the parameter contained by a quantitative map MUST comply
+with the unit description provided for its respective `_suffix` entry.
+For example, a T1 map in milliseconds unit (ms) is not valid, given that the
+description of the `T1map` suffix requires the parameter to be in seconds (s).
+
+#### The `part` entity
+
+This entity shall be used to indicate which component of the complex representation
```

Comment:
Should "shall" here be replaced with one of the RFC2119 terms?  SHALL or MUST? 

Or is it just:
```suggestion
This entity is used to indicate which component of the complex representation
```

Reply:
+1 for "is"

Reply:
@Gilles86 - if you give this a :+1: vote then I think we can go ahead and commit